ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -125,7 +125,42 @@ class LeftNavMenuViewController: UIViewController {
 
     private var statusCell = MSFListCellState()
 
-    private var leftNavMenuSections: [MSFListSectionState] {
+    private var leftNavMenuList = MSFList()
+
+    private var leftNavAccountViewHeightConstraint: NSLayoutConstraint?
+
+    private lazy var leftNavAccountView: UIView = {
+        let chevron = UIImageView(image: UIImage(named: "ic_fluent_ios_chevron_right_20_filled"))
+        chevron.tintColor = Colors.textPrimary
+
+        persona.state.presence = .available
+        persona.state.primaryText = "Kat Larrson"
+        persona.state.secondaryText = "Designer"
+        persona.state.image = UIImage(named: "avatar_kat_larsson")
+        persona.state.titleTrailingAccessoryUIView = chevron
+        persona.state.backgroundColor = .systemBackground
+        persona.state.onTapAction = {
+            self.dismiss(animated: true, completion: {
```

Comment:
Watch out for memory leaks when you capture self in a closure.
The closure will retain a strong reference to the container object (instance of LeftNavMenuViewController) and the instance of LeftNavMenuViewController will retain a strong reference to the closure.

This will prevent these 2 objects from being deallocated for the life of the app.

You can use the snippet below and for multiple operations on the self reference, you can do something like the code in this [link](https://github.com/microsoft/fluentui-apple/blob/b2f6b63cbfd77d6e8bde02e3d9ead05f8d08227d/ios/FluentUI/Bottom%20Commanding/BottomCommandingController.swift#L315):


 ```
    let defaultMenuAction = { [weak self] in
            guard let menuAction = self?.menuAction else {
                return
            }

            menuAction()
        }
```